### PR TITLE
Preparing release for 1.0.0beta12

### PR DESCRIPTION
**Pull request by niezbop**
This is automatically generated to prepare for the new version

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -162,7 +173,6 @@
 - CONTRIBUTING document [\#2](https://github.com/DragonBox/uplift/pull/2) ([lacostej](https://github.com/lacostej))
 - Rakefile improvement: portability using ruby code [\#1](https://github.com/DragonBox/uplift/pull/1) ([lacostej](https://github.com/lacostej))
 
-## [v1.0.0beta1](https://github.com/DragonBox/uplift/tree/v1.0.0beta1) (2017-11-06)
```
Comment (lacostej) [inline CHANGELOG.md]: weird that we lost this

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
 
+**Fixed bugs:**
+
+- LoseVersionRequirement and MinimalVersionRequirement faulty behaviour [\#66](https://github.com/DragonBox/uplift/issues/66)
+
+**Merged pull requests:**
+
+- Introduce Range version requirement [\#67](https://github.com/DragonBox/uplift/pull/67) ([niezbop](https://github.com/niezbop))
+
+## [v1.0.0beta11](https://github.com/DragonBox/uplift/tree/v1.0.0beta11) (2018-12-18)
```
Comment (lacostej) [inline CHANGELOG.md]: weird that the date changed :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -54,6 +64,7 @@
 
 **Implemented enhancements:**
 
+- .gitignore generation with line ending [\#31](https://github.com/DragonBox/uplift/issues/31)
```
Comment (lacostej) [inline CHANGELOG.md]: weird that this appeared.

**Comment by niezbop** [comment]
Yep... That's the reason for #69 so that we can actually get replicability across our releases.

**Pull request merged**